src/globus_cli/commands/flows/run/update.py

```diff
@@ -0,0 +1,108 @@
+from __future__ import annotations
+
+import uuid
+
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import CommaDelimitedList, command, run_id_arg
+from globus_cli.termio import Field, TextMode, display, formatters
+
+
+@command("update", short_help="Update a run")
+@run_id_arg
+@click.option(
+    "--label",
+    type=str,
+    help="A label to give the run.",
+)
+@click.option(
+    "--managers",
+    "run_managers",
```

Comment:
Is there a reason why you adopted an alternate name here but not in the principals for the `flows update` command (I'm guessing we haven't aliased these at the API level...)? No objection, just double checking.

Reply:
Do you mean "not in the principals for the `flow start` command"?

Flows use different terms (administrators, starters, viewers) than Runs (monitors, managers).

As for the difference with `flows start`, I thought the plan was to modify how those commands work, and asap.

Reply:
Sorry, I should have been more precise: In other commands we are not using `flows_` or `run_` prefixes. I was just wondering why this one is.

Reply:
Ah, gotcha.

I did that so it was clearer where that would map to in the API call.

Reply:
I can change that if desired.

Reply:
I don't have a very strong feeling, I just wanted to be sure I understood the intent. This is fine as-is, IMO.